api/config.go

```diff
@@ -53,13 +53,14 @@ type Config struct {
 	BaseKey       []byte
 
 	// Swap configs
-	SwapBackendURL          string         // Ethereum API endpoint
-	SwapEnabled             bool           // whether SWAP incentives are enabled
-	SwapPaymentThreshold    uint64         // honey amount at which a payment is triggered
-	SwapDisconnectThreshold uint64         // honey amount at which a peer disconnects
-	SwapInitialDeposit      uint64         // initial deposit amount to the chequebook
-	SwapLogPath             string         // dir to swap related audit logs
-	Contract                common.Address // address of the chequebook contract
+	SwapBackendURL            string         // Ethereum API endpoint
+	SwapEnabled               bool           // whether SWAP incentives are enabled
+	SwapPaymentThreshold      uint64         // honey amount at which a payment is triggered
+	SwapDisconnectThreshold   uint64         // honey amount at which a peer disconnects
+	SwapInitialDeposit        uint64         // initial deposit amount to the chequebook
+	SwapLogPath               string         // dir to swap related audit logs
+	Contract                  common.Address // address of the chequebook contract
```

Comment:
Not a change introduced in this PR, but I would like `Contract` to be called SwapChequebook.

Two reasons:
1) all other SWAP parameters start with Swap,
2) Contract can be anything. I'd rather tell what kind of contract it is.

Reply:
He actually opened an issue for this himself: https://github.com/ethersphere/swarm/issues/1822

Let's not burden this for this PR but do it as part of another one?

Reply:
I agree, let's keep the old names untouched for now and work on one big rename PR once everybody is back from Japan. Otherwise this PR will explode in number of lines changed.